vm/src/vm.rs

```diff
@@ -563,7 +563,7 @@ impl VirtualMachine {
                 let wrapped = self.call_get_descriptor(func, obj.clone())?;
                 self.invoke(&wrapped, args)
             }
-            None => Err(self.new_type_error(format!("Unsupported method: {}", method_name))),
+            None => Err(self.new_type_error(format!("{:?} object is not callable", cls.name))),
```

Comment:
You should put the change in `vm.invoke` instead, maybe check if it's callable using `vm.is_callable` before calling `call_method`.